bin/create-workflow-templates

```diff
@@ -88,7 +88,29 @@ function inject_script_source {
   echo "${new_template_path}"
 }
 
-# Testing things in parallel
+function install_full_templates {
+  # TODO HARMONY-640 parse out template name instead of using file name
+  f=$1
+  batch_file=$(create_batch_template $f)
+  chain_file=$(parameterize_template $f)
+  workflow_file=$(inject_docker_image $f)
+
+  install_template $workflow_file
+  install_template $batch_file
+  install_template $chain_file
+}
+
+# We install templates in parallel to significantly speed up deployment time.
+# We wait on each of the pids we fork so that we error out if the template
+# creation fails for any of the templates.
+function wait_for_pids {
```

Comment:
I could swear I already made this change once.  Either way, glad you found it wasn't there and did it.

Reply:
I remember seeing this before as well.